slackify-html.js

```diff
@@ -261,6 +261,9 @@ function walk(dom, nesting) {
             var alt = el.attribs.alt;
             out += '<Inline Image' + (alt !== '' ? '('+alt+')' : '') + ': ' + el.attribs.src + '>';
             break;
+          case 'blockquote':
+            out += '>' + walk(el.children) + '\n\n';
```

Comment:
do we need to worry about a blockquote with "\n\n" embedded within it?

Reply:
hm that's an interesting question because if the text has a trailing \n\n at the end, it could be interpreted as new lines in slack, and if that's the case then the blockquote text could be missing the \n\n

Reply:
just tried this in Postman and `<blockquote class="ghq-card-content__block-quote" data-ghq-card-content-type="BLOCK_QUOTE">block quote code snippet\n\n</blockquote>` is transformed with \n\n in the text which looks to be interpreted as text in Slack too

![image](https://user-images.githubusercontent.com/20482216/81185340-741e9000-8f7f-11ea-9351-0c5d5b03eeaa.png)
![image](https://user-images.githubusercontent.com/20482216/81185390-8698c980-8f7f-11ea-831e-cfeb6b9a19b3.png)

Reply:
sorry, i didn't mean those literal characters, i meant, what if the text has a double newline embedded within it.

Reply:
@jahlbornG oh hm do you mind giving an example? it might depend on if it's possible to do so in the webapp or when someone is creating a new card within slack

Reply:
something like:

````````
<blockquote>This is a quote.

with embedded newlines</blockquote>
```````

Reply:
hmmm ok i see. i'm gonna need to follow up with someone who might know more about this. i'm solely relying on the interactions i'm able to make in the webapp/slack and the card html we store in the DB, which currently doesn't make this _seem_ possible. but that of course doesn't mean it's not a thing

Reply:
@jahlbornG Thanks for calling this out! Although I can't recreate it in the webapp, your example and any embedded `<br>` would split the blockquote in the middle. 

I'm going to take a closer look at the html spec for blockquote and modify this PR to account for the possibilities listed here https://html.spec.whatwg.org/multipage/grouping-content.html#the-blockquote-element